crates/libs/implement/src/gen.rs

```diff
@@ -329,11 +401,14 @@ fn gen_query_interface(inputs: &ImplementInputs) -> syn::ImplItemFn {
             interface: *mut *mut ::core::ffi::c_void,
         ) -> ::windows_core::HRESULT {
             unsafe {
+                #base_query
```

Comment:
Feels like we can skip some of the other queries (like the identity) if we have a base query)